pkg/client/core/service_nameserverhandler.go

```diff
@@ -0,0 +1,46 @@
+package core
+
+import (
+	"fmt"
+
+	"github.com/oslokommune/okctl/pkg/client"
+	"github.com/oslokommune/okctl/pkg/client/core/api/git"
+	"github.com/oslokommune/okctl/pkg/github"
+	"github.com/oslokommune/okctl/pkg/spinner"
+)
+
+type nameserverRecordDelegationService struct {
+	spin      spinner.Spinner
+	githubAPI github.Githuber
+}
+
+// CreateNameserverRecordDelegationRequest knows how to request a NS record delegation
```

Comment:
As mentioned in another PR comment (https://github.com/oslokommune/okctl/pull/248/files#diff-887776f525f159dbd372df86c92874e4b36c5e28c0fe920624efffb81b8519fcR31), I'm wondering what a NS record delegation mean. 

Possible godoc trick, use "See also" to avoid repetition: https://elliotchance.medium.com/godoc-tips-tricks-cda6571549b

Reply:
@yngvark do you mean "links"? As far as I can see in the article, only reference to "See also" is for text as a section header

Reply:
OK that was too quick, didn't read properly through that link.

Reply:
But we can put somthing like "// See someotherfile.go".